docker-compose.mongodb.yml

```diff
@@ -15,7 +15,7 @@ services:
 
   mongo:
     image: mongo:4.4.1
-    command: --keyFile=/data/mgkey --replSet=${SKYNET_DB_REPLICASET:-skynet}
+    command: --keyFile=/data/mgkey --replSet=${SKYNET_DB_REPLICASET:-skynet} --setParameter ShardingTaskExecutorPoolMinSize=10
```

Comment:
Hm, why did you only add `ShardingTaskExecutorPoolMinSize`, doesn't the thread also suggest to update `ShardingTaskExecutorPoolMaxConnecting`. It feels a bit "dartsy", like we're just trying something to see what sticks.   

In any case I think changes like this warrant a comment in-line to explain why the parameter is set.

Reply:
Re: `ShardingTaskExecutorPoolMaxConnecting` - I don't quite understand what that option does, to be honest.  

The docs say the following:
```
Maximum number of simultaneous initiating connections (including pending connections in setup/refresh state) each TaskExecutor connection pool can have to a [mongod](https://www.mongodb.com/docs/manual/reference/program/mongod/#mongodb-binary-bin.mongod) instance.
```
That sounds to me as a measure to counter connection depletion and services waiting for initial connection to the cluster. I'm not sure this is relevant? It might be.

> It feels a bit "dartsy", like we're just trying something to see what sticks.
I am afraid that you are spot on. I cannot say with certainty that this will solve the problem.